spec/models/crop_spec.rb

```diff
@@ -597,8 +597,17 @@
   end
 
   context "crop rejections" do
-    let!(:rejected_reason) { FactoryGirl.create(:crop, name: 'tomato', approval_status: 'rejected', reason_for_rejection: 'not edible') }
-    let!(:rejected_other) { FactoryGirl.create(:crop, name: 'tomato', approval_status: 'rejected', reason_for_rejection: 'other', rejection_notes: 'blah blah blah') }
+    let!(:rejected_reason) do
+      FactoryGirl.create(:crop, name: 'tomato',
```

Comment:
i agree on less space - but enforcing this indentation style also makes it clearer that's it's one level - that we're not passing arrays of hashes of arrays. I think we'll catch some existing bugs as we make the contents of long lines come in to view.